src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/SyncCloudContent.targets

```diff
@@ -0,0 +1,31 @@
+<Project ToolsVersion="12.0" DefaultTargets="DownloadBlobsFromAzureTargets" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="DownloadFromAzure" AssemblyFile="$(ToolsDir)net45/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
+  <UsingTask TaskName="ListAzureContainers" AssemblyFile="$(ToolsDir)/net45/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
+
+  <PropertyGroup>
+    <DownloadDirectory Condition="'$(DownloadDirectory)' == ''">$(PackagesDir)/AzureTransfer</DownloadDirectory>
+  </PropertyGroup>
+
+  <Target Name="DownloadBlobsFromAzureTargets" DependsOnTargets="ListAzureContainers;DownloadBlobsFromAzure" />
+
+  <!-- Grab latest azure container if a specific azure container is not specified -->
+  <Target Name="ListAzureContainers" Condition="'$(ContainerName)' == ''">
```

Comment:
The target name may be more appropriate as "GetLatestContainerNameFromAzure" than as "ListAzureContainers".